### PR TITLE
Bump dependencies to support ApiApprovals for .NETStandard

### PR DESCRIPTION
**Pull request by SeanFeldman**
Enable running API approvals against .NET Standard as well.

### CODE REVIEW

**Comment by SeanFeldman** [comment]
Need to iron out the issue with `System.Drawing.Bitmap`.

**Comment by SeanFeldman** [comment]
Removed WIP. Ready to be reviewed.

**Comment by SeanFeldman** [comment]
PR allows API validation on .NET Standard, which was missing before.
@ChristianWolf42 any reason why this can't reviewed and merged?

File: test/Microsoft.Azure.ServiceBus.UnitTests/API/APIApprovals.cs [hunk 1]
```diff
@@ -1,23 +1,26 @@
-﻿#if NET461
+﻿using System;
+using System.Linq;
```
Comment (vinaysurya) [inline test/Microsoft.Azure.ServiceBus.UnitTests/API/APIApprovals.cs]: nit: move into namespace

**Comment by vinaysurya** [comment]
@SeanFeldman, thanks for the PR on this!  Sorry, we are running some hands short in the holiday season and could not get to it earlier. If you can fix the minor nit comment, I can merge the PR.

**Comment by SeanFeldman** [comment]
@vinaysurya done

**Comment by vinaysurya** [comment]
@SeanFeldman, looks like the appveyor build was failing, can you take a look. Also we had to merge the other AAD pack, so can you please refresh your branch.

**Comment by SeanFeldman** [comment]
@vinaysurya failed test didn't look like related to the changes.
I've rebased on top of the latest `dev`. Should be green.

BTW, we can force start tests if needed, no need to wait for the author to re-verify tests are failing or not 🙂

**Comment by SeanFeldman** [comment]
@vinaysurya I can conclude that the changes in this branch have nothing to do with the random failing tests.

Prior to moving 2 using statements under a namespace the tests were green ([build history](https://ci.appveyor.com/project/vinaysurya/azure-service-bus-dotnet/build/2.0.421)).

And then subsequent build has failed a [under Full FW](https://ci.appveyor.com/project/vinaysurya/azure-service-bus-dotnet/build/2.0.429) not as a result of [unit test](https://ci.appveyor.com/project/vinaysurya/azure-service-bus-dotnet/build/2.0.429/job/w2bm0t92hwro429b#L1650).

 This time, there a failing test under .NET Core.
image

Running that failing test locally works and it passes.
image

It feels like flaky tests, or cleanup issue.

Note that the [`dev` branch is failing ATM](https://ci.appveyor.com/project/vinaysurya/azure-service-bus-dotnet/branch/dev). Any PR is a merge of a branch against the target branch and if `dev` has issues, PR will fail tests as well.

**Comment by SeanFeldman** [comment]
Also, the issue is with time these tests execute (#387). Though it has nothing to do with the flakiness.

**Comment by SeanFeldman** [comment]
Glad to see flaky tests addressed @vinaysurya. 🎉

**Pull request merged**